src/cljs/vanilla/widgets/basic_widget.cljs

```diff
@@ -65,8 +79,15 @@
                                                  (rf/dispatch [:remove-widget name])
                                                  (.stopPropagation %))}]]]])
 
-
-(defn basic-widget [name data options custom-content]
+;@TODO - what does data do?
```

Comment:
I believe its the actual data for the graph, may or may not be combined with the highcharts data for the chart at this point